client/pages/torrent/[infoHash].js

```diff
@@ -660,6 +664,20 @@ const Torrent = ({ token, torrent = {}, userId, userRole, uid, userStats }) => {
               : [getLocaleString("torrNo")],
         }}
       />
+      <Infobox mb={5}>
+        <Text fontWeight={600} fontSize={1} textTransform="uppercase" mb={3}>
```

Comment:
Text style is incorrect here, please observe how `textTransform` is applied on the other headings